README.md

```diff
@@ -1,2 +1,48 @@
 # xapi-profiles
 A set of documents addressing the structure of and supporting services for xAPI Profiles.
+
+This Github repository contains the xAPI Profiles Specification. [xAPI](https://github.com/adlnet/xAPI-Spec) is a learning
+technologies interoperability specification that describes communication
+about learner activity and experiences between technologies. The specification stands as a 
+companion to xAPI, and is divided into three documents:
+
+*   Part One:   [About xAPI Profiles](./xapi-profiles-about.md)
+*   Part Two:   [xAPI Profiles Structure](./xapi-profiles-structure.md)
+*   Part Three: [xAPI Profiles Communication](./xapi-profiles-communication.md)
+
+## Specification versions
+
+The current version of the specification is
+[1.0](https://github.com/DataInteroperability/xapi-profiles).
+
+The next version of the specification has not been planned. 
+
+## What to do if the spec is unclear
+
+If when implementing the specification you find something is unclear or
+unhelpful, you can help to improve the xAPI Profiles specification by
+[raising an issue here](https://github.com/DataInteroperability/xapi-profiles/issues). When
```

Comment:
It's possible to create and link to issue templates. I don't know how, but I'm sure Google does. It might be useful to create one based on the items below.